src/DataStructures.jl

```diff
@@ -94,4 +94,7 @@ module DataStructures
     export status
     export deref_key, deref_value, deref, advance, regress
 
+    @deprecate iter Stack
+    @deprecate iter Queue
+
```

Comment:
hm, was looking at the diff here since I've never used this macro, should it just be `@deprecate iter`?